and-bible/app/src/main/java/net/bible/android/view/activity/page/BibleView.java

```diff
@@ -166,8 +166,9 @@ public boolean onJsAlert(WebView view, String url, String message, JsResult resu
 	@Override
 	protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
 		super.onMeasure(widthMeasureSpec, heightMeasureSpec);
-		// update the height in ScreenSettings
-		ScreenSettings.setContentViewHeightPx(getMeasuredHeight());
+		// update the height in ScreenSettings. Global screen height must be taken from parent view
+		// that contains all windows.
+		ScreenSettings.setContentViewHeightPx(((View) getParent().getParent()).getMeasuredHeight());
```

Comment:
Do we need null checks here?

Reply:
I agree this is not elegant. About null checks: with the current layout I think they should never be null and that parent of parent element should be correct. What should we do here anyway if the parent happens to be null (i.e. we reach top level element?)? 

If at some point there is a change in layout structure, this might of course break. Thus, I guess some kind of test would indeed be appropriate. But I'm not (yet) familiar how to implement this kind tests for android.

Reply:
You just compare with null.  Kotlin would be great here.  I can add a couple of null checks.